src/App.tsx

```diff
@@ -63,12 +63,14 @@ function App() {
     )
   }
 
+  const visibleDataSources = ["DemoApplicationDataSource"]
+
   return (
     <ThemeProvider theme={theme}>
       <div>
         <GlobalStyle />
         <ApplicationContext.Provider value={application}>
-          <FSTreeProvider>
+          <FSTreeProvider visibleDataSources={visibleDataSources}>
```

Comment:
Since App.tsx shouldn't be application specific, this kind of config should not be in here, but part of the plugin.
Meaning we can't create the FSTreeprovider in the "bootstrapper"

Reply:
good point, moved the FSTreeProvider stuff to the demo-plugin instead